CoC.txt

```diff
@@ -54,6 +54,76 @@ Members of the community should not hesitate to contact the Community Council if
+================================================================================
+
+As a member of the community, we ask you to:
+
+- Be friendly and patient. Your work will be used by other people, and you in
+  turn will depend on the work of others. Any decision you take will affect
+  human beings, and you should take those consequences into account when making
+  decisions. Remember that our community is international, and that the people
+  you interact with may be communicating with you in a language that is not
+  their native tongue.
+
+- Assume good faith and good intentions. Disagreements — both social and
+  technical — happen all the time. Our diverse backgrounds give us the ability
+  to see things from many different perspectives, but they can also lead to
+  misunderstandings. It is important that we resolve disagreements
+  constructively, and that we do not jump to conclusions about each other's
+  motives.
+
+- Be welcoming, considerate, and respectful. We strive to be a community that
+  welcomes and supports people of all backgrounds and identities. This includes,
+  but is not limited to, members of any race, ethnicity, culture, national
+  origin, color, immigration status, social and economic class, educational
+  level, sex, sexual orientation, gender identity and expression, age, size,
+  family status, political belief, religion and mental and physical ability.
+
+- Be careful in the words you choose. We are a community of professionals, and
+  we conduct ourselves professionally. Exclusionary behavior is absolutely
+  unacceptable. This includes, but is not limited to:
+
+  - Discriminatory (e.g., racist or sexist) jokes and language
+  - Inappropriate names for user accounts, servers, files, repositories, etc.
+  - Posting sexually explicit or violent material
+  - Personal insults
+  - Unwelcome sexual attention
+  - Advocating for, or encouraging, any of the above behavior
+
+- Listen and communicate openly and honestly. Yield the floor to
+  others, and help to make sure that everyone gets heard. In other words, try to
+  be your best self. In doing so, you contribute the health and longevity of
+  this community.
+
+As a member of the SecureDrop community, we ask you to never engage in the
+following behaviors:
+
+- Intimidation, harassments, or insults.  This includes but is not limited to:
+  - Physical intimidation or threats against someone's physical safety
+  - Obscene or intimidating gestures
+  - Stalking
+  - Demeaning another person
+  - Unwelcome following
+  - Enlisting the help of others, whether in person or online, in order to
+    target an individual
+  - Taking photographs, video, or audio recordings or recordings without consent
+  - Shouting
+  - Sustained disruption of talks and events
+  - Disruption of meetings
+  - Inappropriate physical contact
+  - Unwelcome sexual attention
+  - Posting (or threatening to post) other people's personally identifying
+    information ("doxing")
+
```

Comment:
I think this would be a good spot to mention general boundary violations. I know that's kinda vague, but if people need to not be talked to or not be asked any personal questions or not share food or like whatever, there may be a reason. This kinda covers one of the major cases of the "obey the spirit, not just the letter" and might make it easier to enforce the grey areas should be need to. For example.

> Part of having a diverse community withe diverse backgrounds, people you interact with will have different levels of comfort with different topics or social behaviors. Please respect people's stated boundaries, and do not attempt change these boundaries through your actions. This includes repeated requests or demands against a stated boundary.

I think there could be a better way to phrase that, but I'm not exactly an expert on this. :neutral_face:

Reply:
I really appreciate that point, and I think "Respect people's stated personal boundaries" is pretty clear. Unpacking that further risks going more into territory where people want to rules-lawyer about it; I think we can keep it short and sweet.